src/analyses/variable-sensitivity/variable_sensitivity_object_factory.h

```diff
@@ -31,14 +31,29 @@
 #include <util/namespace.h>
 #include <util/options.h>
 
+enum ABSTRACT_OBJECT_TYPET
+{
+  TWO_VALUE,
+  CONSTANT,
+  INTERVAL,
+  ARRAY_SENSITIVE,
```

Comment:
If you want to rename any of these to match the command-line names / make them clearer then I am fine with that.  Likewise, if you want to keep them as-is for clarity, that is also fine.

Reply:
I'll keep them as they are for the moment so we can follow the changes through.